fiber/README.md

```diff
@@ -0,0 +1,127 @@
+<p align="center">
+  <a href="https://sentry.io" target="_blank" align="center">
+    <img src="https://sentry-brand.storage.googleapis.com/sentry-logo-black.png" width="280">
+  </a>
+  <br />
+</p>
+
+# Official Sentry fiber Handler for Sentry-go SDK
+
+**Godoc:** https://godoc.org/github.com/getsentry/sentry-go/fiber
```

Comment:
We should change the URL to `https://pkg.go.dev/github.com/getsentry/sentry-go/fiber` instead

Reply:
Yes, but all the URLs in the repository link to GoDoc. I'll create a separate PR to address that.